code/modules/mob/living/carbon/alien/larva/death.dm

```diff
@@ -0,0 +1,15 @@
+/mob/living/carbon/alien/larva/death(gibbed)
+	if(stat == DEAD)	return

```

Comment:
And this file too

Reply:
I can only assume these were files that were removed  with the buildmode and life/death refactoring.
@Crazylemon64  could confirm.